BB-SC-TC/Testing/BB_Digital_Twin.md

```diff
@@ -0,0 +1,80 @@
+
+# Digital Twin
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC-TC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Testing
```

Comment:
I think digital twins are a lot more broadly applicable than just for Testing

Reply:
Agreed. Will be taken care of in the next pull request.